docs/pages/blog/mui-x-mid-v6-features.md

```diff
@@ -0,0 +1,130 @@
+---
+title: MUI X improvements, Charts in Alpha, Discord, and more!
+description: Support for time zones, a new filtering UI, charts and Discord
+date: 2023-07-30T00:00:00.000Z
+authors: ['richbustos', 'josefreitas']
+card: true
+tags: ['MUI X', 'News']
+---
+
+<img src="/static/blog/mui-x-mid-v6-features/intro.png" alt="" width="2400" height="600" style="margin-bottom: 16px;" />
+
+Since the first v6 stable release, we have continuously rolled out new major features.
+Now that we reached [MUI X v6.10.x](https://github.com/mui/mui-x/releases), it's time to delve into the most recent additions to MUI X v6, and why you should get the latest version now.
+
+## Table of contents
+
+- [Date Pickers](#date-pickers)
+  - [Support for Time zones](#support-for-time-zones)
+  - [Digital Clock](#digital-clock)
+- [Data Grid](#data-grid)
+  - [Copy and Paste](#copy-and-paste)[<span class="plan-premium"></span>](/x/introduction/licensing/#premium-plan)
+  - [Filter on Column headers](#filter-on-column-headers)[<span class="plan-pro"></span>](/x/introduction/licensing/#pro-plan)
+  - [Performance Enhancements: Smoother Scrolling and Expanding Rows](#performance-enhancements-smoother-scrolling-and-expanding-rows)
```

Comment:
I think we should revert the order, MIT before Pro before Premium.

Reply:
What's the perspective on this one?

Reply:
If the ultimate goal is to convert community users to paid users, then I'd lead with the Premium and Pro features.

Reply:
I made the changes to MIT, Pro, then Premium as requested. 

I am a fan of Gary V's book, "[Jab, Jab, Jab, Right Hook](https://www.amazon.com/Jab-Right-Hook-Story-Social/dp/006227306X)".

This is essentially a marketing tactic for "give the people what they want for free, then hit them with a sales pitch" lol